### DISCUSSION TITLE
About the user experience

### DISCUSSION THREAD

**Discussion by MrYZhou** [fastapi/fastapi#12980] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
app = FastAPI(docs_url=None)

.vscode - launch.json

   "module": "fastapi", 
      "args": ["run", "main.py","--host", "127.0.0.1", "--port", "8000","--reload"],
```

### Description

I start with fastapi-cli and I see the document address, but my address is set to empty. I can't use it when I open it

![image](https://github.com/user-attachments/assets/5c344b1c-af72-43af-babb-1635f1288f67)

### Operating System

Windows

### FastAPI Version

0.115.5

### Pydantic Version

2.10.1

### Python Version

3.13

**Comment by YuriiMotov** [comment]
There is already issue in FastAPI CLI repo: https://github.com/fastapi/fastapi-cli/issues/162
And PR: https://github.com/fastapi/fastapi-cli/pull/119

Let's track it there
